vassal-app/src/main/java/VASSAL/build/module/Map.java.orig

```diff
@@ -0,0 +1,3499 @@
+/*
```

Comment:
This file shouldn't have been committed.

Reply:
Oh I see it was THAT day.